hw/ip/clkmgr/data/clkmgr.hjson.tpl

```diff
@@ -232,11 +232,11 @@
         },
         {
           bits: "7:4",
-          name: "LOW_SPEED_SEL",
+          name: "HI_SPEED_SEL",
           mubi: true,
           desc: '''
-            A value of kMultiBitBool4True selects low speed external clocks.
-            All other values selects nominal speed clocks.
+            A value of kMultiBitBool4True selects nominal speed external clocks.
```

Comment:
This doesn't seem quite right. My understanding is that the external clock frequency is always the same, but this controls the io clkmgr dividers.

Also, this makes the mubi protected configuration one where the dividers are disabled, meaning the corresponding io clocks run faster. I guess that means the unprotected config of external clocks is as when lc requests external clocks?

Reply:
actually the external clock frequencies can change.
We are expecting a combination of 48MHz and 96MHz.

So when its 96MHz, we need to set this to "true", so that the dividers remain at nominal values.
When it is 48MHz, we need to set this to "false", so that the dividers remain at half values, and the output clock frequency is the same.  Let me know if that logic seems roughly right to you. 

I'll double check the behavior just to make sure.